### ISSUE TITLE
Missing Home Indication on Home Page Title

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#127]
Screengrab/videos (insert file names here)
N/A

Link
https://d8-stg.boston.gov

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
JAWS, Voice Over, Talkback

Trusted Tester Test

Brief Description
Title tag on homepage just says Boston.gov. Add “Home” to the title tag to provide meaningful indication that user is on the homepage. 

Steps to reproduce
1)	Press insert + T simultaneously to read current page title. 
2)	Alternately, open page source code and examine title tag in <head>

Desired Result
User would expect some kind of indicator other than the web address to know they are on the homepage. “Home – Boston.gov” for instance.

Actual Result
There is no kind of indicator other than the web address to tell the user if they are on the homepage.

**Comment by subaha-cob** [comment]
@andrebtlima What exactly do we want the title of the Homepage to be? I am going with "Home – Boston.gov" for now. Let me know otherwise.

**Comment by subaha-cob** [comment]
@andrebtlima I had to put in a temporary fix. I am not completely sure it is not just working the way the other pages are working. For fix, i added the title to the Page title of the homepage. That got picked up by [node:title]. I didn't need to do that for the other pages. They worked right. I will keep my eye out for why the homepage is the only not working correctly.

**Comment by subaha-cob** [comment]
Testing at: https://d8-uat.boston.gov/

**Comment by jill-iterators** [comment]
The Home page title is currently "boston.gov", this should be changed to "Home-Boston.gov" in order to indicate this is the Home page.

**Comment by jill-iterators** [comment]
Retested in Incognito, this is fixed.

**Comment by subaha-cob** [comment]
@jill-iterators and @david-iterators please remember all testing should be done on our UAT environment. 

Testing at: https://d8-uat.boston.gov/
